code/__DEFINES/loadout.dm

```diff
@@ -437,7 +437,7 @@ GLOBAL_LIST_INIT(medic_clothes_listed_products, list(
 		/obj/item/armor_module/storage/general = list(CAT_MOD, "General Purpose Storage Module", 0, "black"),
 		/obj/item/armor_module/storage/engineering = list(CAT_MOD, "Engineering Storage Module", 0, "black"),
 		/obj/item/armor_module/storage/grenade = list(CAT_MOD, "Grenade Storage Module", 0, "black"),
-		/obj/item/storage/pouch/medical_injectors/medic = list(CAT_POU, "Advanced Autoinjector pouch", 0, "orange"),
+		/obj/item/storage/pouch/medical_injectors/basic = list(CAT_POU, "Combat autoinjector pouch", 0, "black"),
```

Comment:
I don't get why you're replacing the special autoinjector pouch specifically for corpsmen with one meant for standard marines.

Reply:
> for the same reason as all the others, it gives free gear that isnt included with other pouches that the medic gets, the whole point of this PR. meaning that you are given a best pouch option even if you arent taking injectors. The two solutions are either making all the stuff contained within the advanced autoinjector pouch standard or removing it and considering that the contents of this pouch are:
> 
> 3 advanced combat autoinjectors - 15 points 1 dexplus autoinjector - not in basic 2 QK+ autoinjectors - 2 points 2 Peri+ autoinjectors - 2 points
> 
> I decided to play it safe since they can just buy the stuff normally

That's the point though, they're a corpsmen. The medkit pouch does the same thing giving them burn and brute kits alongside a meraderm hypospray. The basic autoinjector pouch is also entirely useless for medics as they're medical belts have all those chems already there in larger amounts. 
The argument doesn't make sense for a pop limited support class that's supposed to have advanced medical items and medical skills. Especially since it's a suggested pouch that's supposed to give every medic no matter their loadout important medicines they need to do their job, this just harms newer medics and people buying expensive things like mera/derm or an extra evac bed as they won't be able to get meds they need to do their job.
That argument is honestly kind of roundabout and I can ask why they get their belts for free then if that gives them limited items which aren't standard.